test_requirements.txt

```diff
@@ -1,4 +1,4 @@
-black==20.8b1
+black==22.3.0
```

Comment:
Updated black version as the issue `ImportError: cannot import name '_unicodefun' from 'click'` has been fixed by black's 22.3.0 version as stated [here](https://github.com/psf/black/issues/2964)